recipes/amptk/meta.yaml

```diff
@@ -8,10 +8,10 @@ package:
 source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: https://github.com/nextgenusfs/{{ name }}/archive/{{ version }}.tar.gz
-  sha256: ba8fb8ba0bb440863031613b7ce2074ab356abbdf5931280e582261f4a0e30cc
+  sha256: 54c109237fc06bda36bb788b5ae1069593171f4a36db037a078e6736e04292cc
```

Comment:
why has the checksum changed?

Reply:
I had a slight typo in code.

Reply:
This is not good practice. You don't change the version, but in the results can change. Better is to do released and minor releases.

Reply:
If you decrease the build number again to 0, I will remove the previous version from anaconda.org